.github/workflows/deploy.yml

```
@@ -101,6 +103,16 @@ jobs:
           SSH_HOST: ${{ secrets.SSH_HOST }}
           DST_PATH: ${{ secrets.DST_PATH }}
 
+      - name: DB Migration
+        run: >
+          ssh -i ${{ runner.temp }}/key -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${SSH_USER}@${SSH_HOST} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)